### ISSUE TITLE
Seeing this on DEDI a17 stable

### ISSUE THREAD

**Issue by logancautrell** [dmustanger/7dtd-ServerTools#146]
**Describe the bug**
Console log.

**How To Reproduce**
Don't know it was in the log.

**Expected behavior**
No NRE

**Screenshots**
```2019-01-03T09:06:23 31798.411 ERR [MODS] Error while executing EntityKilled on mod "Server Tools"
2019-01-03T09:06:23 31798.411 EXC Object reference not set to an instance of an object
NullReferenceException: Object reference not set to an instance of an object
  at ServerTools.API.EntityKilled (.Entity _entity1, .Entity _entity2) [0x00000] in <filename unknown>:0 
  at ModEvent`2[Entity,Entity].Invoke (.Entity _a1, .Entity _a2) [0x00000] in <filename unknown>:0 
UnityEngine.DebugLogHandler:Internal_LogException(Exception, Object)
UnityEngine.DebugLogHandler:LogException(Exception, Object)
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:LogException(Exception)
Logger:masterLogException(Exception)
Logger:Exception(Exception)
Log:Exception(Exception)
ModEventAbs`1:LogError(Exception, Receiver)
ModEvent`2:Invoke(Entity, Entity)
EntityAlive:OnEntityDeath()
EntityZombie:OnEntityDeath()
EntityAlive:ClientKill_Patch1(Object, DamageResponse)
EntityAlive:Kill(DamageResponse)
EntityAlive:ProcessDamageResponseLocal(DamageResponse)
EntityAlive:damageEntityLocal(DamageSource, Int32, Boolean, Single)
EntityAlive:DamageEntity(DamageSource, Int32, Boolean, Single)
EntityEnemy:DamageEntity(DamageSource, Int32, Boolean, Single)
EntityZombie:DamageEntity(DamageSource, Int32, Boolean, Single)
EntityAlive:OnUpdateEntity()
World:TickEntities(Single)
GameManager:gmUpdate()
GameManager:Update()
 
(Filename:  Line: -1)
```

**Desktop (please complete the following information):**
 - OS: OS: Windows 7  (6.1.0) 64bit -
 Dedi from ping perfect
 - ServerTools Version 12.5

**Additional context**
None

UPLOAD A COMPLETE LOG FILE WITH YOUR BUG REPORT! IF THERE IS NO LOGFILE WE WILL CLOSE THE REPORT!

[output_log-3.txt](https://github.com/dmustanger/7dtd-ServerTools/files/2724506/output_log-3.txt)

**Comment by albiejames** [comment]
This problem was addressed in version 12.7

**Comment by logancautrell** [comment]
Ok thanks. Will update.
